### PR TITLE
fix Add font resize option to item page

### PR DESCRIPTION
**Pull request by Inna-r**
issue #292

The problem was "share item" element's missing closing tag.
Fixed for mjs and story pages as well.
@TheGrandVizier  Font resize option was added to all 3 html item page templates (depending on collection): image, video and text (the rest).
The default font size for each item page is 18px, so the image and video item page's font was changed  accordingly.

### CODE REVIEW

File: scss/main/_item.scss [hunk 1]
```diff
@@ -129,69 +110,56 @@
...
                             font-size: 20px;
                             margin-bottom: 8px;
                         }
-
-                        &__view, &__remove {
+                        &__view,
+                        &__remove {
                             text-decoration: underline;
-                            pointer: cursor;
+                            cursor: pointer;
```
Comment (OriHoch) [inline scss/main/_item.scss]: this looks like a different change from the resize font - if it was intended, please document it

File: scss/main/_item.scss [hunk 2]
```diff
@@ -286,83 +244,70 @@
...
-                                    &:hover {
-                                        opacity: 0.5;
+                                    .he & {
+                                        transform: translate(-50%, -50%) skew(16deg);
                                     }
                                 }
-
+                                &:hover {
+                                    opacity: 0.5;
+                                }
```
Comment (OriHoch) [inline scss/main/_item.scss]: looks like &:hover moved to different place - need to make sure it doesn't have negative / unrelated / unexpected effects

File: scss/main/_item.scss [hunk 3]
```diff
@@ -434,220 +373,184 @@
...
                         .pictures-info {
                             display: flex;
                             position: absolute;
                             top: -346px;
                             line-height: 1.5;
                             color: #999999;
                             width: 94px;
-                            font-size: 15px;
-
+                            font-size: 18px;
```
Comment (OriHoch) [inline scss/main/_item.scss]: looks like design change on item page - should be documented

File: scss/main/_item.scss [hunk 4]
```diff
@@ -434,220 +373,184 @@
...
-                  }
-
+                    position: relative;
+                    padding: 26px 0;
+                    .en & {
+                        margin-left: 26px;
+                    }
+                    .he & {
+                        margin-right: 26px;
+                    }
```
Comment (OriHoch) [inline scss/main/_item.scss]: lot's of changes here - not sure if they are actual changes or just whitespace - need to make sure it's only whitespace changes
if it's actual design changes - need to document it

File: scss/main/_item.scss (hunk 2 above)
Comment (Inna-r) [inline scss/main/_item.scss]: &:hover wasn't really moved, still has the same location in file

File: scss/main/_item.scss (hunk 4 above)
Comment (Inna-r) [inline scss/main/_item.scss]: The only real change here is the class name

**Pull request merged**